eva-accession-import-dbsnp2/src/test/resources/application.properties

```diff
@@ -1,9 +1,10 @@
 spring.batch.job.names=IMPORT_DBSNP_JSON_VARIANTS_JOB
 
 # dbSNP JSON input source
-parameters.input=src/test/resources/input-files/test-dbsnp.json.bz2
+parameters.input=src/test/resources/input-files/json/test-dbsnp.json.bz2
```

Comment:
How many input files do we have? Because if it's just the 2 (one JSON and one assembly report), I think it may not be worth creating the subfolders.